admin_guide/out_of_resource_handling.adoc

```diff
@@ -12,39 +12,120 @@ toc::[]
+in the xref:out-of-resource-eviction-policy[node configuration file] or through the xref:out-of-resource-eviction-policy-cli[node command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. Review the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+
+By using well-configured eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource. 
+
+[NOTE]
+====
+When the node fails a pod, it terminates all containers in the pod, and
+the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
+====
+
+[[out-of-resource-create-config]]
+=== Using the Node Configuration to Create a Policy 
+
+To configure an eviction policy using the node configuration file:
 
-Using eviction policies, a node can proactively monitor for and prevent
-against total starvation of a compute resource.
+. Edit the node configuration file on a node.
 
-In cases where a node is running low on available resources, it can proactively
-fail one or more pods in order to reclaim the starved resource using an eviction
-policy. When the node fails a pod, it terminates all containers in the pod, and
-the `PodPhase` is transitioned to *Failed*.
+. Add or edit the eviction thresholds under the `eviction-hard` or `eviction-soft` paramters, using the following examples
+as general guidelines and not recommended settings:
++
+For example:
++
+.Sample Node Configuration file for a hard eviction 
+====
+----
+kubeletArguments:
+  eviction-hard: <1>
+  - memory.available<500Mi<2>
+  - nodefs.available<500Mi
+  - nodefs.inodesFree<100Mi
+  - imagefs.available<100Mi
+  - imagefs.inodesFree<100Mi
+----
+
+<1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[hard eviction].
+<2> An eviction threshold based on a specific eviction trigger signal.
+====
++
+.Sample Node Configuration file for a soft eviction
+====
+----
+kubeletArguments:
+  eviction-soft: <1>
+  - memory.available<500Mi <2>
+  - nodefs.available<500Mi
+  - nodefs.inodesFree<100Mi
+  - imagefs.available<100Mi
+  - imagefs.inodesFree<100Mi
+  eviction-soft-grace-period:<3>
+  - memory.available=1m30s
+  - nodefs.available=1m30s
+  - nodefs.inodesFree=1m30s
+  - imagefs.available=1m30s
+  - imagefs.inodesFree=1m30s
+----
+
+<1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[soft eviction].
+<2> An eviction threshold based on a specific eviction trigger signal.
+<3> The grace period for the soft eviction. Leave the default values for best performance.
```

Comment:
Changed to "optimal performance." Search in Git found a couple "optimal performance" and one "best... performance", but no "ideal performance."